cmd/kubeadm/app/constants/constants.go

```diff
@@ -366,6 +366,34 @@ const (
 	// May be overridden by a flag at startup.
 	// Deprecated: use the secure KubeControllerManagerPort instead.
 	InsecureKubeControllerManagerPort = 10252
+
+	// IPVSProxyMode is match set up cluster with ipvs proxy model
+	// TODO (yastij): we should make this shared also with componentConfig
+	IPVSProxyMode = "ipvs"
```

Comment:
Ideally these consts should be made part of the kube-proxy versioned API. I'll push for that in the next versions of it, but for now, this is a fine solution.